apps/health_campaign_field_worker_app/lib/blocs/facility/facility.dart

```diff
@@ -71,6 +77,11 @@ class FacilityBloc extends Bloc<FacilityEvent, FacilityState> {
       final facility = facilities
           .firstWhereOrNull((element) => element.id == latestStock?.facilityId);
 
+      allFacilities.sort((a, b) => b.auditDetails!.lastModifiedTime
+          .compareTo(a.auditDetails!.lastModifiedTime));
+      facilities.sort((a, b) => b.auditDetails!.lastModifiedTime
```

Comment:
@vishal-egov  add a null check

Reply:
Moved to a different place and added null check there